src/prisma/schema.prisma

```diff
@@ -4067,3 +4069,19 @@ model BlockchainLog {
   @@index([contractId])
   @@index([eventName])
 }
+
+model GithubIssue {
+  id            String      @id @default(uuid()) @db.Uuid
+  pullRequestId String
+  githubEventId String      @db.Uuid
+  githubEvent   GithubEvent @relation(fields: [githubEventId], references: [id], onDelete: Cascade)
+  repoId        Int
+  repo          GithubRepo  @relation(fields: [repoId], references: [id], onDelete: Cascade)
+  issueNumber   Int
+  tags          Json        @default("[]")
```

Comment:
what's the structure of tags? if it's just al ist of strings, you should use String[]